.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+cache: pip
+
+env:
+  global:
+    - PYTHONPATH="./tests/"
+    - DJANGO_SETTINGS_MODULE="settings_test"
+
+matrix:
+  include:
+    - python: 3.6
+    # Workaround for Python 3.7 support. See:
+    # https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+# Workaround for testing Boto3 on Travis. Taken from:
+# https://github.com/azavea/django-amazon-ses/blob/57ed8619435194742308c6f15cbf9c91b231b88a/.travis.yml#L17-L19
+before_install: sudo rm /etc/boto.cfg
+
+install: pip install .[tests]
+
+script: django-admin test --noinput
+
+deploy:
+  - provider: pypi
+    user: django-ecsmanage
+    skip_existing: true
+    server: https://test.pypi.org/legacy/
+    password:
+      secure: 0x32eNc+iKoYxM+nvMEEQJIoPHq3ehNqVUyKMT6crntYBBC8xgD2NHwsF0HCtfgm/NitWFlZLwtPhpb8dxTW//hh5gafkLt4/MALHFYBeRYm0AihCfTkjiuC/FCkpQ9K7X9dVO5qQN5gi5PQvkFW13NrKWOUCMTr2/jE8mLaRS3exCKUUamsd/S6BxgXY7LHm50ueDFDTYSkZrqKwIHnwwpMLcLvZZxcRJXZxpK567e8KTpcBelG+/ecY+Cn/+L0JJ29p6CGWCuW+vaIeYaeHAU6XZZ/cqNuycYlFTX+CpKSlajQgFQTMC1O1L6OVX73gSpZ2jUSd5dSimDmxgttsoo+kCxPA/hY9cUfXUpS1c52xVv9ZXZgweIV0g/EXcKLe1SIal8wy6fTE5DFzyZsZutE6ipksZjgrsFM26IHPflsf/caM/n3nDqoeXW5FtA8t4wpsemYWRFnYhCbWUWpiqAweKzzK/+3AJKEf55d4Zk7TokEjg5bZdBLroI0zy4MM6clxzujZNpn8FgTtlKhN9sDtHU5AvSe0+BUJRHJQVVSbc0wk5L+03dYarPlhShIWISSjoLFgDJhJPCZLgWD1OTetv0IekxU38AJnyqUi5XvUUbtOOiyNyU6xCFVp2XMQdRwMv41hYvEFbY4Su1B3AF2vSJCvK1HQehvzAMt4JE=
+    on:
+      distributions: sdist bdist_wheel
+      repo: azavea/django-ecsmanage
+      branch: develop
+  - provider: pypi
+    user: django-ecsmanage
+    skip_existing: true
+    password:
+      secure: 0x32eNc+iKoYxM+nvMEEQJIoPHq3ehNqVUyKMT6crntYBBC8xgD2NHwsF0HCtfgm/NitWFlZLwtPhpb8dxTW//hh5gafkLt4/MALHFYBeRYm0AihCfTkjiuC/FCkpQ9K7X9dVO5qQN5gi5PQvkFW13NrKWOUCMTr2/jE8mLaRS3exCKUUamsd/S6BxgXY7LHm50ueDFDTYSkZrqKwIHnwwpMLcLvZZxcRJXZxpK567e8KTpcBelG+/ecY+Cn/+L0JJ29p6CGWCuW+vaIeYaeHAU6XZZ/cqNuycYlFTX+CpKSlajQgFQTMC1O1L6OVX73gSpZ2jUSd5dSimDmxgttsoo+kCxPA/hY9cUfXUpS1c52xVv9ZXZgweIV0g/EXcKLe1SIal8wy6fTE5DFzyZsZutE6ipksZjgrsFM26IHPflsf/caM/n3nDqoeXW5FtA8t4wpsemYWRFnYhCbWUWpiqAweKzzK/+3AJKEf55d4Zk7TokEjg5bZdBLroI0zy4MM6clxzujZNpn8FgTtlKhN9sDtHU5AvSe0+BUJRHJQVVSbc0wk5L+03dYarPlhShIWISSjoLFgDJhJPCZLgWD1OTetv0IekxU38AJnyqUi5XvUUbtOOiyNyU6xCFVp2XMQdRwMv41hYvEFbY4Su1B3AF2vSJCvK1HQehvzAMt4JE=
+    on:
```

Comment:
Given the matrix build configuration, I believe that Travis will try to publish the module twice (one for 3.6 and one for 3.7), unless we do something like: https://github.com/azavea/django-amazon-ses/blob/develop/.travis.yml#L41.